### ISSUE TITLE
Retain tablespace when compressing chunk

### ISSUE THREAD

**Issue by erimatnor** [timescale/timescaledb#2016]
Compressing a chunk should not have the side-effect of moving the data to another tablespace. If a user wants to move a chunk to another tablespace, irrespective of whether it is compressed or not, this should happend via `ALTER TABLE chunk SET TABLESPACE space1` on the original chunk. Setting the tablespace on the auxiliary compressed data chunk should not be allowed (nor should other ALTER commands for that matter).

In other words, we should retain the invariant that the empty original chunk and the internal compressed chunk should always live in the same tablespace and changing it should happen via the original chunk. This properly retains the abstraction of a single chunk (which can be compressed or not) and retains one way to interact with the abstract chunk.

**Comment by erimatnor** [comment]
@pmwkaa can we close this issue?

**Comment by pmwkaa** [comment]
Implemented in: https://github.com/timescale/timescaledb/commit/add97fbadfd3d4d73953fa436390e2fee157badf and https://github.com/timescale/timescaledb/commit/fca7e36898652f177e45aa454ad985cb381c6567
